README.md

```diff
@@ -1,25 +1,49 @@
-# kubevirt-ansible
+# KubeVirt Ansible
 
 This repository provides collection of playbooks to
-* install KubeVirt on existing OpenShift cluster
-* deploy K8S on given machines and install KubeVirt
-* deploy OpenShift on given machines and install KubeVirt
+- [x] [install KubeVirt on existing OpenShift cluster](#install-kubevirt-on-existing-openshift-cluster)
+- [ ] deploy Kubernetes cluster on given machines and install KubeVirt
+- [x] [deploy OpenShift cluster on given machines and install KubeVirt](#deploy-kubernetes-or-openshift-and-kubevirt)
+- [ ] deploy Kubernetes cluster with KubeVirt with Lago
+- [x] [deploy Kubernetes cluster with KubeVirt with Lago](#deploy-new-kubernetes-or-openshift-cluster-and-kubevirt-with-lago)
 
-Tested on CentOS Linux release 7.3.1611 (Core), OpenShift 3.7 and Ansible 2.3.1
+**Tested on CentOS Linux release 7.4 (Core), OpenShift 3.7 and Ansible 2.4.2**
 
+## List of playbooks
+
+| Playbook name | Description |
+| ------------- | ----------- |
+| install-kubevirt-on-openshift.yml | This playbook installs KubeVirt on existing OpenShift cluster. |
+| deploy-kubernetes.yml | This playbook deploys Kubernetes cluster on given machines. |
```

Comment:
Does it deploy kubernete sonly? Or including kubevirt?

Reply:
only kubernetes cluster without kubevirt